src/shell/server.cpp

```diff
@@ -387,7 +387,7 @@ void server::run() {
        We added this hack because the json library code used for ensuring this property
        was crashing when compiling Lean on Windows with mingw. */
 #if !defined(LEAN_EMSCRIPTEN)
-    std::setlocale(LC_NUMERIC, "C");
+    setlocale(LC_NUMERIC, "C");
 #endif
```

Comment:
(For a range of commits last week, I couldn't build because std::setlocale wasn't being resolved.)